rumps/rumps.py

```diff
@@ -981,8 +1031,8 @@ def fallbackOnName(self):
             self.nsstatusitem.setTitle_(self._app['_name'])
 
     @classmethod
-    def callback_(cls, nsmenuitem):
-        self, callback = cls._ns_to_py_and_callback[nsmenuitem]
+    def callback_(cls, sender):
+        self, callback = cls._ns_to_py_and_callback[sender]
```

Comment:
Let's revert `sender` back to `nsmenuitem`, as we're not using type annotation in rumps at this point in time.